app/interactors/creates_company/create_owner.rb

```diff
@@ -0,0 +1,21 @@
+module CreatesCompany
+  class CreateOwner
+    include Interactor
+
+    delegate :company, :user, to: :context
+
+    def call
+      raise_error unless member.save
+    end
+
+    private
+
+    def member
+      CompanyMember.new(company: company, user: user, role: :owner)
+    end
+
+    def raise_error
+      context.fail!
+    end
+  end
+end
```

Comment:
It's a good practice to return an error when we fail context. What do you think about it?

```suggestion
module CreatesCompany
  class CreateOwner
    include Interactor

    delegate :company, :user, to: :context

    def call
      context.fail!(error: "Company can't be created.") if member.invalid?
    end

    private

    def member
      CompanyMember.create(company: company, user: user, role: :owner)
    end
  end
end
```